api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/EditN3GeneratorVTwo.java

```diff
@@ -78,8 +79,14 @@ public void subInMultiUris(Map<String,List<String>> varsToVals, List<String> n3t
         }         
     }
   
-    private boolean containsNullOrEmpty(List<String> values) {        
-        return values != null && ( values.contains(null) || values.contains("") );       
+    private boolean containsNullOrEmpty(List<String> values) {
+    	//return values != null && ( values.contains(null) || values.contains("") );
+    	if(values == null) return true ;
+    	if(values.isEmpty()) return true ;
```

Comment:
Another minor note, any reason not to merge this into a single check?

`if(values == null || values.isEmpty()) return true ;`